src/components/NewsRoomCard.jsx

```diff
@@ -0,0 +1,58 @@
+import {
+  Card,
+  CardContent,
+  CardFooter,
+  Button
+} from "@baltimorecounty/dotgov-components";
+import React from "react";
+
+const NewsRoomCard = props => {
+  const { publishDate, title, author, articleSummary, thumbnail, url } = props;
+
+  const options = {
+    year: "numeric",
+    month: "long",
+    day: "numeric"
+  };
+
+  const published = new Date(publishDate).toLocaleDateString("en-US", options);
+
+  return (
+    <Card className="text-left">
+      <CardContent>
+        <div class="row">
+          <h3>{title}</h3>
+        </div>
+        <div className="d-flex col-12">
+          <div class="row">
+            <div id="dg_main-content">
+              <img
+                className="dg_image left"
+                src="//baltimorecountymd.gov/sebin/t/t/homepage-county-executive.jpg"
```

Comment:
the src here should use "thumbnail" we are going. We are also going to have conditional logic if it doesn't exist.

So the scenario thumbnail doesn't exist you will display this icon

```html
<i class="far fa-newspaper fa-5x"></i>
```
Then I think you will have `75px` top and bottom padding and `70px` left and right. Should have blue background and should jsut be about a square i think.

![image](https://user-images.githubusercontent.com/1933400/74438929-30613200-4e39-11ea-88fc-1c251426d9f9.png)

Reply:
we will need to check with sam, but i think we probably will need to come up with some rules around thumbnails.

Reply:
this is done but the sizing is odd and we will definitely need clarification on how it should react to different sizes.